src/serverMain/kotlin/com/studo/campusqr/endpoints/LocationManagement.kt

```diff
@@ -161,6 +163,14 @@ suspend fun ApplicationCall.visitLocation() {
   respondOk()
 }
 
+suspend fun AuthenticatedApplicationCall.guestVisitLocation() {
+  if (user.type == UserType.ACCESS_MANAGER || user.isAdmin) {
```

Comment:
what about moderators? when you ask for user.isAccessManager then i think we don't have this problem